Contrib doesnt build on Linux gcc (ICU)

When attempting to build contrib with gcc it throws ICU related errors:

```console

/usr/bin/gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer -std=c++0x -fno-canonical-system-headers -Wno-builtin-macro-redefined -D__DATE__=redacted -D__TIMESTAMP__=redacted -D__TIME__=redacted -DABSL_MIN_LOG_LEVEL=4 -fdebug-types-section -fPIC -Wno-deprecated-declarations -std=c++17 -fPIC -DU_CHARSET_IS_UTF8=1 -DU_USING_ICU_NAMESPACE=0 -DUCONFIG_ONLY_HTML_CONVERSION=1 -DUCONFIG_NO_LEGACY_CONVERSION=1 -DUCONFIG_NO_BREAK_ITERATION=1 -DUCONFIG_NO_COLLATION=1 -DUCONFIG_NO_FORMATTING=1 -DUCONFIG_NO_TRANSLITERATION=1 -DUCONFIG_NO_REGULAR_EXPRESSIONS=1 -W -Wall -pedantic -Wpointer-arith -Wwrite-strings -Wno-long-long   -fuse-ld=lld -Wl,-no-as-needed -Wl,-z,relro,-z,now -B/usr/bin -pass-exit-codes -lm -fuse-ld=gold -l:libstdc++.a -Wl,--gc-sections   -o ../../bin/makeconv gencnvex.o genmbcs.o makeconv.o ucnvstat.o -L../../lib -licutu -L../../lib -licui18n -L../../lib -licuuc -L../../stubdata -licudata -lpthread -lm
                                                                            makeconv.o:makeconv.cpp:DW.ref.__gxx_personality_v0: error: undefined reference to '__gxx_personality_v0'                                                                         ../../lib/libicuuc.a(umutex.ao):umutex.cpp:function std::mutex::lock(): error: undefined reference to 'std::__throw_system_error(int)'                                            ../../lib/libicuuc.a(umutex.ao):umutex.cpp:function std::once_flag::_Prepare_execution::~_Prepare_execution(): error: undefined reference to 'std::__once_callable'               ../../lib/libicuuc.a(umutex.ao):umutex.cpp:function std::once_flag::_Prepare_execution::~_Prepare_execution(): error: undefined reference to 'std::__once_call'                   ../../lib/libicuuc.a(umutex.ao):umutex.cpp:function umtx_cleanup: error: undefined reference to 'std::condition_variable::~condition_variable()'                                  ../../lib/libicuuc.a(umutex.ao):umutex.cpp:function umtx_init::{lambda()#2}::operator()() const: error: undefined reference to 'std::condition_variable::condition_variable()'    ../../lib/libicuuc.a(umutex.ao):umutex.cpp:function icu_72::umtx_initImplPreInit(icu_72::UInitOnce&): error: undefined reference to 'std::condition_variable::wait(std::unique_lock<std::mutex>&)'                                                                                                                                                                  ../../lib/libicuuc.a(umutex.ao):umutex.cpp:function icu_72::umtx_initImplPostInit(icu_72::UInitOnce&): error: undefined reference to 'std::condition_variable::notify_all()'      ../../lib/libicuuc.a(umutex.ao):umutex.cpp:function void std::call_once<void (&)()>(std::once_flag&, void (&)()): error: undefined reference to '__once_proxy'                    ../../lib/libicuuc.a(umutex.ao):umutex.cpp:function void std::call_once<void (&)()>(std::once_flag&, void (&)()): error: undefined reference to 'std::__throw_system_error(int)'  ../../lib/libicuuc.a(umutex.ao):umutex.cpp:function std::once_flag::_Prepare_execution::_Prepare_execution<std::call_once<void (&)()>(std::once_flag&, void (&)())::{lambda()#1}>(void (&)())::{lambda()#1}::operator()() const: error: undefined reference to 'std::__once_callable'                                                                               ../../lib/libicuuc.a(umutex.ao):umutex.cpp:function std::once_flag::_Prepare_execution::_Prepare_execution<std::call_once<void (&)()>(std::once_flag&, void (&)())::{lambda()#1}>(
void (&)()): error: undefined reference to 'std::__once_callable'                                                                                                                 
../../lib/libicuuc.a(umutex.ao):umutex.cpp:function std::once_flag::_Prepare_execution::_Prepare_execution<std::call_once<void (&)()>(std::once_flag&, void (&)())::{lambda()#1}>(
void (&)()): error: undefined reference to 'std::__once_call'                                                                                                                     
../../lib/libicuuc.a(umutex.ao):umutex.cpp:function std::unique_lock<std::mutex>::lock(): error: undefined reference to 'std::__throw_system_error(int)'                          
../../lib/libicuuc.a(umutex.ao):umutex.cpp:function std::unique_lock<std::mutex>::lock(): error: undefined reference to 'std::__throw_system_error(int)'                          
collect2: error: ld returned 1 exit status                                                                                                                                        
make[2]: *** [Makefile:81: ../../bin/makeconv] Error 1                                                                                                                            make[2]: Leaving directory '/b/f/w/bazel-out/k8-fastbuild/bin/bazel/foreign_cc/unicode_icu_build.build_tmpdir/tools/makeconv'                                                     
make[1]: *** [Makefile:47: all-recursive] Error 2                                                                                                                                 
make[1]: Leaving directory '/b/f/w/bazel-out/k8-fastbuild/bin/bazel/foreign_cc/unicode_icu_build.build_tmpdir/tools'                                                              
make: *** [Makefile:153: all-recursive] Error 2  
```

I think I have a similar issue related to CEL upgrade when building on OSX, I've tried mac 11.7.10 and 12.7.1 
AWS ami: 

ami-0da052d2225304b8f | amzn-ec2-macos-12.7.1-20231220-195030
ami-06bf601fad285cc69 | amzn-ec2-macos-11.7.10-20230922-233531

```
Undefined symbols for architecture x86_64:
  "cel::interop_internal::ProtoStructValueToMessageWrapper(cel::Value const&)", referenced from:
      cel::interop_internal::ToLegacyValue(google::protobuf::Arena*, cel::Handle<cel::Value> const&, bool) in libinterop.a(interop.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
Target //contrib/exe:envoy-static failed to build
```

Both didn't work for me. Last time I used macos-11.7.10 to build 1.28.0 and had no issues. 

```
Apple clang version 13.0.0
```

seems its failing on all release branches - altho not sure if different error - this is 1.26

```console
   gcc	 ...  /b/f/w/external/com_github_unicode_org_icu/icu4c/source/tools/makeconv/ucnvstat.c
/usr/bin/gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer -std=c++0x -fno-canonical-system-headers -Wno-builtin-macro-redefined -D__DATE__=redacted -D__TIMESTAMP__=redacted -D__TIME__=redacted -DABSL_MIN_LOG_LEVEL=4 -fPIC -Wno-deprecated-declarations -std=c++17 -fPIC -DU_CHARSET_IS_UTF8=1 -DU_USING_ICU_NAMESPACE=0 -DUCONFIG_ONLY_HTML_CONVERSION=1 -DUCONFIG_NO_LEGACY_CONVERSION=1 -DUCONFIG_NO_BREAK_ITERATION=1 -DUCONFIG_NO_COLLATION=1 -DUCONFIG_NO_FORMATTING=1 -DUCONFIG_NO_TRANSLITERATION=1 -DUCONFIG_NO_REGULAR_EXPRESSIONS=1 -W -Wall -pedantic -Wpointer-arith -Wwrite-strings -Wno-long-long   -fuse-ld=lld -Wl,-no-as-needed -Wl,-z,relro,-z,now -B/usr/bin -pass-exit-codes -lm -fuse-ld=gold -l:libstdc++.a -Wl,--gc-sections   -o ../../bin/makeconv gencnvex.o genmbcs.o makeconv.o ucnvstat.o -L../../lib -licutu -L../../lib -licui18n -L../../lib -licuuc -L../../stubdata -licudata -lpthread -lm  
makeconv.o:makeconv.cpp:DW.ref.__gxx_personality_v0: error: undefined reference to '__gxx_personality_v0'
../../lib/libicuuc.a(umutex.ao):umutex.cpp:function std::mutex::lock(): error: undefined reference to 'std::__throw_system_error(int)'
../../lib/libicuuc.a(umutex.ao):umutex.cpp:function umtx_cleanup: error: undefined reference to 'std::condition_variable::~condition_variable()'
../../lib/libicuuc.a(umutex.ao):umutex.cpp:function umtx_init::{lambda()#2}::operator()() const: error: undefined reference to 'std::condition_variable::condition_variable()'
../../lib/libicuuc.a(umutex.ao):umutex.cpp:function icu_72::umtx_initImplPreInit(icu_72::UInitOnce&): error: undefined reference to 'std::condition_variable::wait(std::unique_lock<std::mutex>&)'
../../lib/libicuuc.a(umutex.ao):umutex.cpp:function icu_72::umtx_initImplPostInit(icu_72::UInitOnce&): error: undefined reference to 'std::condition_variable::notify_all()'
../../lib/libicuuc.a(umutex.ao):umutex.cpp:function std::call_once<void (&)()>(std::once_flag&, void (&)())::{lambda()#2}::operator()() const: error: undefined reference to 'std::__once_callable'
../../lib/libicuuc.a(umutex.ao):umutex.cpp:function void std::call_once<void (&)()>(std::once_flag&, void (&)()): error: undefined reference to 'std::__once_callable'
../../lib/libicuuc.a(umutex.ao):umutex.cpp:function void std::call_once<void (&)()>(std::once_flag&, void (&)()): error: undefined reference to 'std::__once_call'
../../lib/libicuuc.a(umutex.ao):umutex.cpp:function void std::call_once<void (&)()>(std::once_flag&, void (&)()): error: undefined reference to '__once_proxy'
../../lib/libicuuc.a(umutex.ao):umutex.cpp:function void std::call_once<void (&)()>(std::once_flag&, void (&)()): error: undefined reference to 'std::__throw_system_error(int)'
../../lib/libicuuc.a(umutex.ao):umutex.cpp:function std::unique_lock<std::mutex>::lock(): error: undefined reference to 'std::__throw_system_error(int)'
../../lib/libicuuc.a(umutex.ao):umutex.cpp:function std::unique_lock<std::mutex>::lock(): error: undefined reference to 'std::__throw_system_error(int)'
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:81: ../../bin/makeconv] Error 1
make[2]: Leaving directory '/b/f/w/bazel-out/k8-fastbuild/bin/bazel/foreign_cc/unicode_icu_build.build_tmpdir/tools/makeconv'
make[1]: *** [Makefile:47: all-recursive] Error 2
make[1]: Leaving directory '/b/f/w/bazel-out/k8-fastbuild/bin/bazel/foreign_cc/unicode_icu_build.build_tmpdir/tools'
make: *** [Makefile:153: all-recursive] Error 2
make: Leaving directory '/b/f/w/bazel-out/k8-fastbuild/bin/bazel/foreign_cc/unicode_icu_build.build_tmpdir'
_____ END BUILD LOGS _____
rules_foreign_cc: Build wrapper script location: bazel-out/k8-fastbuild/bin/bazel/foreign_cc/unicode_icu_build_foreign_cc/wrapper_build_script.sh
rules_foreign_cc: Build script location: bazel-out/k8-fastbuild/bin/bazel/foreign_cc/unicode_icu_build_foreign_cc/build_script.sh
rules_foreign_cc: Build log location: bazel-out/k8-fastbuild/bin/bazel/foreign_cc/unicode_icu_build_foreign_cc/Configure.log
```

@florincoras Your suggestion helped to move past this linking issue. I now getting other errors that, I think, indicate a different issue (see below). These errors suggest that some symbols missing in the VPP extension, but I saw similar issues with other Envoy modules when compiled with gcc. In other cases they manifested when I used `-c opt`, while other configurations work fine. So I think it's not really a VPP specific issue anymore as much as some issue when compiling optimized binaries with gcc.
I think it manifests for VPP because it by default builds optimized binaries, regardless of the bazel configuration AFAIU.

```
SUBCOMMAND: # //contrib/exe:envoy-static [action 'Linking contrib/exe/envoy-static', configuration: efdfc2f379a3cae84691392e64ad2267e6fe30be4337824d25da362874352e1b, execution platform: @local_config_platform//:
host]                                                                                                                                                                                                              
(cd /mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/execroot/envoy && \                                                                                                                        
  exec env - \                                                                                                                                                                                                     
    BAZEL_COMPILER=gcc \                                                                                                                                                                                           
    BAZEL_LINKLIBS=-l%:libstdc++.a \                                                                                                                                                                               
    BAZEL_LINKOPTS=-lm \                                                                                                                                                                                           
    CC=gcc \                                                                                                                                                                                                       
    CXX=g++ \                                                                                                                                                                                                      
    PATH=/bin:/usr/bin:/usr/local/bin \                                                                                                                                                                            
    PWD=/proc/self/cwd \                                                                                                                                                                                           
  /usr/bin/gcc @bazel-out/k8-fastbuild/bin/contrib/exe/envoy-static-2.params)                                                                                                                                      
# Configuration: efdfc2f379a3cae84691392e64ad2267e6fe30be4337824d25da362874352e1b                                                                                                                                  
# Execution platform: @local_config_platform//:host                                                                                                                                                                
ERROR: /mount/data/ws/envoy/envoy/contrib/exe/BUILD:31:16: Linking contrib/exe/envoy-static failed: (Exit 1): gcc failed: error executing command (from target //contrib/exe:envoy-static) /usr/bin/gcc @bazel-out/
k8-fastbuild/bin/contrib/exe/envoy-static-2.params                                                                                                                                                                                                                                                                                                                                                                                    
Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging                                                                                                       /mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:594: error: undefined referen
ce to 'svm_msg_q_lock_and_alloc_msg_w_ring'                                                                                                                                                                        /mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:597: error: undefined referen
ce to 'svm_msg_q_msg_data'                                                                                                                                                                                         /mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:605: error: undefined referen
ce to 'svm_msg_q_add_and_unlock'                                                                                                                                                                                   /mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:594: error: undefined referen
ce to 'svm_msg_q_lock_and_alloc_msg_w_ring'                                                                                                                                                                        /mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:597: error: undefined referen
ce to 'svm_msg_q_msg_data'                                                                                                                                                                                         /mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:605: error: undefined referen
ce to 'svm_msg_q_add_and_unlock'                                                                                                                                                                                   /mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:594: error: undefined referen
ce to 'svm_msg_q_lock_and_alloc_msg_w_ring'                                                                                                                                                                        /mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:597: error: undefined referen
ce to 'svm_msg_q_msg_data'                                                                                                                                                                                         /mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:605: error: undefined referen
ce to 'svm_msg_q_add_and_unlock'                                                                                                                                                                                   /mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:649: error: undefined referen
ce to 'svm_msg_q_alloc_msg_w_ring'                                                                                                                                                                                 /mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:650: error: undefined referen
ce to 'svm_msg_q_msg_data'                                                                                                                                                                                         /mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:653: error: undefined referen
ce to 'svm_msg_q_add_and_unlock'                                                                                                                                                                                   /mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:648: error: undefined referen
ce to 'svm_msg_q_or_ring_wait_prod'                                                                                                                                                                                /mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:594: error: undefined referen
ce to 'svm_msg_q_lock_and_alloc_msg_w_ring'                                                                                                                                                                        /mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:1171: error: undefined reference to 'svm_msg_q_sub'  
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:1178: error: undefined reference to 'svm_msg_q_free_msg'                                                                                                                                                                                                                
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:34: error: undefined reference to 'svm_msg_q_sub_raw_batch'                                                                                                                                                                                                             
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:2567: error: undefined reference to 'svm_msg_q_free_msg'                                                                                                                                                                                                                
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:34: error: undefined reference to 'svm_msg_q_sub_raw_batch'                                                                                                                                                                                                             
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:1227: error: undefined reference to 'svm_msg_q_free_msg'                                                                                                                                                                                                                
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:789: error: undefined reference to 'svm_fifo_dequeue'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:2075: error: undefined reference to 'svm_ms[135/2198]
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:754: error: undefined referen
ce to 'svm_fifo_peek'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:762: error: undefined referen
ce to 'svm_fifo_peek'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:764: error: undefined referen
ce to 'svm_fifo_peek'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:769: error: undefined referen
ce to 'svm_fifo_dequeue_drop'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:2075: error: undefined reference to 'svm_msg_q_wait'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:649: error: undefined referen
ce to 'svm_msg_q_alloc_msg_w_ring'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:648: error: undefined referen
ce to 'svm_msg_q_or_ring_wait_prod'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:787: error: undefined referen
ce to 'svm_fifo_peek'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:1407: error: undefined reference to 'fifo_segment_mai
n_init'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:1803: error: undefined reference to 'svm_msg_q_wait'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:2192: error: undefined reference to 'svm_fifo_segment
s'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:2187: error: undefined reference to 'svm_msg_q_wait'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:2229: error: undefined reference to 'svm_fifo_dequeue
_drop'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:683: error: undefined referen
ce to 'svm_fifo_enqueue_segments'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:710: error: undefined referen
ce to 'svm_fifo_enqueue'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:649: error: undefined referen
ce to 'svm_msg_q_alloc_msg_w_ring'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:648: error: undefined referen
ce to 'svm_msg_q_or_ring_wait_prod'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:683: error: undefined referen
ce to 'svm_fifo_enqueue_segments'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:710: error: undefined referen
ce to 'svm_fifo_enqueue'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:649: error: undefined referen
ce to 'svm_msg_q_alloc_msg_w_ring'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:648: error: undefined referen
ce to 'svm_msg_q_or_ring_wait_prod'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:710: error: undefined referen
ce to 'svm_fifo_enqueue'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vnet/session/application_interface.h:683: error: undefined referen
ce to 'svm_fifo_enqueue_segments'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:4392: error: undefined reference to 'svm_msg_q_sub'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:4396: error: undefined reference to 'svm_msg_q_free_m
sg'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:34: error: undefined reference to 'svm_msg_q_sub_raw_
batch'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:34: error: undefined reference to 'svm_msg_q_sub_raw_
batch'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:3325: error: undefined reference to 'svm_msg_q_timedw
ait'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vppcom.c:2555: error: undefined reference to 'svm_msg_q_timedw
ait'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:404: error: undefined reference to 'vl_msg_api_allo
c'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:415: error: undefined reference to 'vl_msg_api_send
_shmem'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/bazel-out/k8-opt-exec-2B5CBBC6/bin/contrib/vcl/source/build.build_tmpdir/CMakeFiles/vnet/session/sessio
n.api.h:968: error: undefined reference to 'vl_api_string_len'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/bazel-out/k8-opt-exec-2B5CBBC6/bin/contrib/vcl/source/build.build_tmpdir/CMakeFiles/vnet/sessi[74/2198]
n.api.h:332: error: undefined reference to 'vl_api_string_len'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/bazel-out/k8-opt-exec-2B5CBBC6/bin/contrib/vcl/source/build.build_tmpdir/CMakeFiles/vnet/session/sessio
n.api.h:333: error: undefined reference to 'vl_api_format_string'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/bazel-out/k8-opt-exec-2B5CBBC6/bin/contrib/vcl/source/build.build_tmpdir/CMakeFiles/vnet/session/sessio
n.api.h:920: error: undefined reference to 'vl_api_string_len'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/bazel-out/k8-opt-exec-2B5CBBC6/bin/contrib/vcl/source/build.build_tmpdir/CMakeFiles/vnet/session/sessio
n.api.h:229: error: undefined reference to 'vl_api_string_len'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/bazel-out/k8-opt-exec-2B5CBBC6/bin/contrib/vcl/source/build.build_tmpdir/CMakeFiles/vnet/session/sessio
n.api.h:230: error: undefined reference to 'vl_api_format_string'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/bazel-out/k8-opt-exec-2B5CBBC6/bin/contrib/vcl/source/build.build_tmpdir/CMakeFiles/vnet/session/sessio
n.api_fromjson.h:295: error: undefined reference to 'vl_api_c_string_to_api_string'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/bazel-out/k8-opt-exec-2B5CBBC6/bin/contrib/vcl/source/build.build_tmpdir/CMakeFiles/vnet/session/sessio
n.api_fromjson.h:112: error: undefined reference to 'vl_api_c_string_to_api_string'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:297: error: undefined reference to 'vl_client_get_f
irst_plugin_msg_id'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:317: error: undefined reference to 'vl_msg_api_conf
ig'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:317: error: undefined reference to 'vl_msg_api_conf
ig'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:317: error: undefined reference to 'vl_msg_api_conf
ig'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:317: error: undefined reference to 'vl_msg_api_conf
ig'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vlibapi/api_common.h:398: error: undefined reference to 'my_api_ma
in'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:503: error: undefined reference to 'socket_client_m
ain'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:505: error: undefined reference to 'vl_client_api_u
nmap'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vlibmemory/memory_client.h:77: error: undefined reference to 'my_m
emory_client_main'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:532: error: undefined reference to 'vl_socket_clien
t_connect2'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:542: error: undefined reference to 'vl_socket_clien
t_init_shm2'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:126: error: undefined reference to 'vl_api_from_api
_to_new_c_string'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:84: error: undefined reference to 'vl_socket_client
_recv_fd_msg2'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:96: error: undefined reference to 'vl_api_from_api_
to_new_c_string'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:110: error: undefined reference to 'svm_msg_q_set_e
ventfd'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:220: error: undefined reference to 'vl_api_from_api
_to_new_c_string'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:185: error: undefined reference to 'vl_socket_clien
t_recv_fd_msg2'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:197: error: undefined reference to 'vl_api_from_api
_to_new_c_string'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:211: error: undefined reference to 'svm_msg_q_set_e
ventfd'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:351: error: undefined reference to 'vl_msg_api_allo
c'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:381: error: undefined reference to 'vl_msg_api_send
_shmem'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:378: error: undefined reference to 'vl_api_vec_to_a
pi_string'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:389: error: undefined reference to 'vl_msg_api_allo
c'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:395: error: undefined reference to 'vl_msg_api_send
_shmem'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:570: error: undefined reference to 'vl_soc[12/2198]
t_disconnect2'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:572: error: undefined reference to 'vl_client_disco
nnect_from_vlib'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:329: error: undefined reference to 'vl_msg_api_allo
c'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:336: error: undefined reference to 'vl_msg_api_send
_shmem'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:720: error: undefined reference to 'vl_client_send_
disconnect'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_bapi.c:728: error: undefined reference to 'vl_socket_clien
t_recv_fd_msg2'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_cfg.c:298: error: undefined reference to 'vl_set_memory_ui
d'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_cfg.c:303: error: undefined reference to 'vl_set_memory_gi
d'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:436: error: undefined reference to 'fifo_segment
_attach'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:477: error: undefined reference to 'fifo_segment
_get_segment'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:478: error: undefined reference to 'fifo_segment
_delete'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:543: error: undefined reference to 'fifo_segment
_get_segment'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:544: error: undefined reference to 'fifo_segment
_alloc_fifo_w_offset'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:545: error: undefined reference to 'fifo_segment
_alloc_fifo_w_offset'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:551: error: undefined reference to 'fifo_segment
_get_segment'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:552: error: undefined reference to 'fifo_segment
_msg_q_attach'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:585: error: undefined reference to 'fifo_segment
_get_segment_if_valid'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:590: error: undefined reference to 'fifo_segment
_free_client_fifo'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:591: error: undefined reference to 'fifo_segment
_free_client_fifo'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:594: error: undefined reference to 'fifo_segment
_get_segment_if_valid'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:599: error: undefined reference to 'fifo_segment
_free_client_fifo'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:600: error: undefined reference to 'fifo_segment
_free_client_fifo'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:623: error: undefined reference to 'fifo_segment
_get_segment'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:624: error: undefined reference to 'fifo_segment
_msg_q_attach'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:647: error: undefined reference to 'fifo_segment
_msg_qs_discover'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:672: error: undefined reference to 'fifo_segment
_alloc_chunk_w_slice'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:673: error: undefined reference to 'fifo_segment
_chunk_offset'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:689: error: undefined reference to 'fifo_segment
_duplicate_fifo'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:690: error: undefined reference to 'fifo_segment
_duplicate_fifo'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:694: error: undefined reference to 'svm_fifo_add
_subscriber'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_private.c:695: error: undefined reference to 'svm_fifo_add
_subscriber'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_sapi.c:93: error: undefined reference to 'svm_msg_q_set_ev
entfd'
/mount/data/bazel/cache/_bazel_kmu/ca4c8634fe188ae2080ff00715ba1313/sandbox/linux-sandbox/35/execroot/envoy/external/com_github_fdio_vpp_vcl/src/vcl/vcl_sapi.c:246: error: undefined reference to 'svm_msg_q_set_e
ventfd'
collect2: error: ld returned 1 exit status                                                               
Target //contrib/exe:envoy-static failed to build                                                        
Use --verbose_failures to see the command lines of failed build steps.                                                                                                                                             
INFO: Elapsed time: 3429.496s, Critical Path: 1579.28s                                                   
INFO: 12412 processes: 3157 internal, 9253 linux-sandbox, 1 local, 1 worker.                                                                                                                                       
FAILED: Build did NOT complete successfully
```

I took one of the functions referenced in the linker error and tried to see where it's coming from and where it should defined to get some anchors in the code to start from:

```
bazel-out/k8-opt/bin/external/v8/_objs/v8_libshared_noicu/macro-assembler-shared-ia32-x64.o(.debug_addr+0x180): error: undefined reference to 'v8::internal::Assembler::vpsllw(v8::internal::XMMRegister, v8::inter
nal::XMMRegister, v8::internal::XMMRegister)'
```

I could actually find a reference to this function in the code: https://github.com/v8/v8/blob/bb6560830541d9d937e0ae2e2164d8708362fcae/src/codegen/shared-ia32-x64/macro-assembler-shared-ia32-x64.cc#L473.

What I could not find though is where the function is implemented. I know that the version of instruction with 3 registers exists in Intel specs and I could find in V8 code the version of this instruction with 2 registers and an immediate. However, I could not find any place that defines a version with 3 registers.

It's not super easy to search through V8 code generation code because it's somewhat macro heavy, so to double check I did the following:

```
cat bazel-out/k8-opt/bin/contrib/exe/envoy-static-2.params | grep 'bazel-out/k8-opt/.*[.]o' | xargs nm -C 2>/dev/null | grep -B10 -A10 'vpsllw'
```

that also didn't yield the right symbol in any of the object files that were referenced in the linker command (it did find a version of vpsllw with 2 registers and an immediate, but not the version with 3 registers - the same thing I had after looking through the V8 code).

So unless I made a mistake somewhere, the linking error might actually be legit in the sense that this function is referenced and does not exist. Because the issue only manifests when building with `-c opt` it looks like one of the following has to be true:

1. Non-optimized build somehow does define the right version of `vpsllw`
2. Non-optimized build does not actually need a version of `vpsllw` - somehow fastbuild manages to optimzie the thing away, while the optimized build does not
3. Bazel just does not include all the files that are needed in the final linker command line

I'm going to repeat the same thing I did above for fastbuild to check if somehow maybe options 1 or 2 are true with `nm` tool next. And if it turns out that the right version of `vpsllw` acturally exists in the non-optimized code, then I will at least know where it's defined and can check if the object file exists in optimized builds.

--- a/.bazelrc
+++ b/.bazelrc
@@ -57,9 +57,9 @@ test --experimental_ui_max_stdouterr_bytes=11712829 #default 1048576
 # Allow tags to influence execution requirements
 common --experimental_allow_tags_propagation
 
+build:linux --copt=-fdebug-types-section
 # Enable position independent code (this is the default on macOS and Windows)
 # (Workaround for https://github.com/bazelbuild/rules_foreign_cc/issues/421)
-build:linux --copt=-fdebug-types-section
 build:linux --copt=-fPIC
 build:linux --copt=-Wno-deprecated-declarations
 build:linux --cxxopt=-std=c++20 --host_cxxopt=-std=c++20
@@ -95,6 +95,21 @@ build:gcc --linkopt=-fuse-ld=gold --host_linkopt=-fuse-ld=gold
 build:gcc --test_env=HEAPCHECK=
 build:gcc --action_env=BAZEL_COMPILER=gcc
 build:gcc --action_env=CC=gcc --action_env=CXX=g++
+# This is to work around a bug in GCC that makes debug-types-section
+# option not play well with fission:
+# https://gcc.gnu.org/bugzilla/show_bug.cgi?id=110885
+build:gcc --copt=-fno-debug-types-section
+# These trigger errors in multiple places both in Envoy dependecies
+# and in Envoy code itself when using GCC.
+# And in all cases the reports appear to be clear false positives.
+build:gcc --copt=-Wno-error=restrict
+build:gcc --copt=-Wno-error=uninitialized
+build:gcc --cxxopt=-Wno-missing-requires
+# We need this because -Wno-missing-requires options is rather new
+# in GCC, so flags -Wno-missing-requires exists in GCC 12, but does
+# not in GCC 11 and GCC 11 is what is used in docker-gcc
+# configuration currently
+build:gcc --cxxopt=-Wno-unknown-warning
 
 # Clang-tidy
 # TODO(phlax): enable this, its throwing some errors as well as finding more issues
@@ -374,6 +389,7 @@ build:docker-clang-libc++ --config=docker-sandbox
 build:docker-clang-libc++ --config=rbe-toolchain-clang-libc++
 
 build:docker-gcc --config=docker-sandbox
+build:docker-gcc --config=gcc
 build:docker-gcc --config=rbe-toolchain-gcc
 
 build:docker-asan --config=docker-sandbox

--- a/bazel/envoy_internal.bzl
+++ b/bazel/envoy_internal.bzl
@@ -68,7 +68,21 @@ def envoy_copts(repository, test = False):
                    "-Wc++2a-extensions",
                    "-Wrange-loop-analysis",
                ],
-               repository + "//bazel:gcc_build": ["-Wno-maybe-uninitialized"],
+               repository + "//bazel:gcc_build": [
+                   "-Wno-maybe-uninitialized",
+                   # GCC implementation of this warning is too noisy.
+                   #
+                   # It generates warnings even in cases where there is no ambiguity
+                   # between the overloaded version of a method and the hidden version
+                   # from the base class. E.g., when the two have different number of
+                   # arguments or incompatible types and therefore a wrong function
+                   # cannot be called by mistake without triggering a compiler error.
+                   #
+                   # As a safeguard, this warning is only disabled for GCC builds, so
+                   # if Clang catches a problem in the code we would get a warning
+                   # anyways.
+                   "-Wno-error=overloaded-virtual",
+               ],
                # Allow 'nodiscard' function results values to be discarded for test code only
                # TODO(envoyproxy/windows-dev): Replace /Zc:preprocessor with /experimental:preprocessor
                # for msvc versions between 15.8 through 16.4.x. see

